File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq = new PriorityQueue<>(); 
        for (int x : nums) {
            pq.offer(x); 
        }
        while (pq.size() > k) {
            pq.poll(); 
        }
        return pq.peek(); 
    }
}

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.py
import heapq
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        heapq.heapify(nums)
        
        k = len(nums) - k

        for i in range(k):
            heapq.heappop(nums)

        return heapq.heappop(nums)